src/RcppExports.cpp

```diff
@@ -163,6 +163,10 @@ BEGIN_RCPP
 END_RCPP
 }
 
+
+// export this function so it can be called at the C level by other packages:
+int _RcppCCTZ_getOffset(long long s, const char* tzstr);
```

Comment:
`int64_t`, maybe, or `uint64_t`, or `...int128_t` ?  `long long` is less determined across architectures methinks.

Also: do we really need more than, say `uint64_t` ?

Reply:
Indeed, I pulled that from older code; I have changed that to `std::int_fast64_t` as that is what's used throughout the library.

Reply:
I tend to use plain `int64_t` more but I trust the CCTZ folks on this :)